src/sections/Careers/Careers-Programs-single/index.js

```diff
@@ -1,25 +1,41 @@
 import React from "react";
-import {Link} from "gatsby";
+import Select from "react-select";
+import { Link } from "gatsby";
 import { MDXRenderer } from "gatsby-plugin-mdx";
 import { IoIosArrowDropleftCircle } from "react-icons/io";
-import { Row, Container } from "../../../reusecore/Layout";
+import { Row, Container, Col } from "../../../reusecore/Layout";
 import PageHeader from "../../../reusecore/PageHeader";
+import ProgramsPageWrapper, {dropdownTheme, selectStyles} from "./ProgramsSingle.style.js";
 
-import ProgramsPageWrapper from "./ProgramsSingle.style.js";
+const ProgramsSingle = ({ data, options, setActiveOption, activeOption }) => {
+
+  const { frontmatter, body } = data;
 
-const ProgramsSingle = ({data}) => {
-  const { frontmatter, body } = data.mdx;
   return (
     <ProgramsPageWrapper>
       <Container>
-        <Row>
-          <div className="backBtn">
-            <Link to="/careers/programs">
-              <IoIosArrowDropleftCircle />
-              <h4>Go Back</h4>
-            </Link>
-          </div>
-        </Row>
+        <div className="rowWrapper">
```

Comment:
This class can be added to the `Row` component itself and the CSS can be applied to it accordingly. No need for an extra div.

Reply:
Done